CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 #### Visual Environment
 
 - [Delete key will delete selected nodes][1538].
+- [It is possible to move around after deleting a selected visualization][1556].
```

Comment:
what does it mean "deleting a [...] visualization"? It is possible to delete a node with an attached visualization, but it is not possible to delete a visualization per see. Moreover, this should be described much more user friendly / in more explanatory way, like "There was a bug that after a node with a selected visualization was deleted, it was impossible to pan the stage. The error is fixed now."